### PR TITLE
v1.1.1 -> v1.2.0 changes

### PR DESCRIPTION
**Pull request by classicskids**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,48 +23,102 @@
             <groupId>org.esa.snap</groupId>
             <artifactId>ceres-core</artifactId>
 			<version>${snap.version}</version>
+			<exclusions>
```
Comment (marpet) [inline pom.xml:26]: Why are you excluding the tools artifact? This should actually not cause any problem when using OpenJDK 8.

**Review by marpet — COMMENTED** [review]
I only wonder about the exclusion of the tools artifact. The other changes look fine. Even though I can't judge them thematically.

**Comment by classicskids** [comment]
This is code I was given (v1.2.0), and is being used by multiple users. 

I'll be creating a branch for v1.2.1 and I will remove the tools artifact exclusion then.

Many thanks

File: pom.xml (hunk 1 above)
Comment (classicskids) [reply pom.xml:26]: This is code I was given (v1.2.0), and is being used by multiple users.

I'll be creating a branch for v1.2.1 and I will remove the tools artifact exclusion then.

Many thanks

**Pull request merged**